diff --git a/index.js b/index.js
index f6ceaac923694a9e458247b5b28553fed91c4182..2c330fb069391148d8907554cb6cdfdf4b869610 100644
--- a/index.js
+++ b/index.js
@@ -7,6 +7,7 @@ exports.getAllSubResources = void 0;
 const puppeteer_1 = __importDefault(require("puppeteer"));
 async function* getAllSubResources(url, options = {}) {
     let caughtError;
+    console.log(process.env.PUPPETEER_EXECUTABLE_PATH)
     const browser = await puppeteer_1.default.launch();
     try {
         const page = await browser.newPage();
